metricbeat/module/ceph/_meta/config.yml

```diff
@@ -0,0 +1,25 @@
+- module: ceph
+  metricsets: ["perf","status","df","osdpoolstats"]
```

Comment:
Perhaps we can split up the PR in multiple parts to have one metricset first and the others to follow. This will make reviews easier and should help to get a first working PR in faster.